catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -792,25 +791,24 @@ private void generateMetacardAndContentItems(StorageRequest storageRequest,
                 try {
```

Comment:
How about opening the `InputStream` here rather than nesting another `try` block?